gradle-plugin/build.gradle

```diff
@@ -5,6 +5,7 @@ plugins {
     id "java-gradle-plugin"
     id "groovy"
     id "maven-publish"
+    id "jacoco"
```

Comment:
Does that mean we shouldn't ignore the failing code coverage :upside_down_face: ?

Reply:
At least that was my attempt.
But sonarqube does not seem to pick up any coverage for the plugin code :disappointed: